csv/stringify.ts

```diff
@@ -219,23 +219,99 @@ function getValuesFromItem(
 /**
  * Converts an array of objects into a CSV string.
  *
- * @example Usage
+ * @example default options
+ * ```ts
+ * import { stringify } from "@std/csv/stringify";
+ * import { assertEquals } from "@std/assert/equals";
+ *
+ * const data = [
+ *   ["Rick", 70],
+ *   ["Morty", 14],
+ * ];
+ *
+ * assertEquals(stringify(data), `Rick,70\r\nMorty,14\r\n`);
+ * ```
+ *
+ * @example an array of objects and specify columns
```

Comment:
Could you please tweak the example titles to have the correct casing, like those in #5605?

Reply:
Done :)